chpl/chpl-service/src/test/java/gov/healthit/chpl/web/controller/CertifiedProductControllerTest.java

```diff
@@ -441,6 +438,204 @@ public void test_updatePendingCertifiedProductDTO_icsAndRetiredTTs_warningvsErro
 	}
 	
 	/** 
+	 * GIVEN A user edits a certified product
+	 * WHEN they view their pending products
+	 * THEN they should see errors if any privacy and security framework values do not match one of "Approach 1", "Approach 2", "Approach 1;Approach 2"
+	 * (Note: Validation should be generous with case and whitespace)
```

Comment:
This validation is not "generous with case and whitespace" with respect to whitespace in the middle of the string. As in, `Approach 1; Approach 2` should be valid, though coerced to the canonically correct `Approach 1;Approach 2`. I think it is also reasonable to accept commas instead of semi-colons, though coerce them to semi-colons for the final value as well.